Mage.Client/src/main/java/mage/client/deckeditor/DeckImportClipboardDialog.java

```diff
@@ -68,7 +68,13 @@ private Optional<String> getClipboardStringData() {
     }
 
     private void onOK() {
-        tmpPath = DeckUtil.writeTextToTempFile(editData.getText());
+        String decklist = editData.getText();
+        decklist = decklist.replace(FORMAT_TEXT, "");
+        if (decklist.startsWith("Deck\n")) {
+            tmpPath = DeckUtil.writeTextToTempFile("cbimportdeck", ".mtga", decklist);
+        } else {
+            tmpPath = DeckUtil.writeTextToTempFile(decklist);
+        }
```

Comment:
Dumb question, but what is this for?
Could use a descriptive comment for someone not familiar with this part of the code.

Reply:
I added comments there about DeckImportClipboardDialog accepting .mtga lists and how a list is detected to be in .mtga format.